### ISSUE TITLE
[DataCap Application] BITRISE CAPITAL 

### ISSUE THREAD

**Issue by Bitrise2020** [filecoin-project/filecoin-plus-large-datasets#139]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: BITRISE CAPITAL FOUNDATION PTE.LTD
- Website / Social Media: https://www.bitrisevc.com/
- Total amount of DataCap being requested (between 500 TiB and 5 PiB): 5 PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB): 100 TiB
- On-chain address for first allocation: f1dfkdmjhuvvol6okun57mix447wdpolwcd323ktq

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Bitrise Capital is a venture capital institution focusing on the blockchain field. Bitrise’s main investment direction is in the field of digital currency and mining. Bitrise Capital has invested and incubated more than 100 projects.
Bitrise Capital is aimed to provide safe and reliable decentralization financial services. What’s more, we providing asset management, digital asset payment, and connecting DeFi ecology, NFT ecology, Polkadot ecology and many other track. Bitrise has serving 50+ institutions, covering multiple countries, and managing mainstream encrypted digital assets over $500 millions. In Bitrise Capital, you can experience a variety of native asset trading strategies and high-return strategies with in-depth research on multiple Defi projects, and provide various asset value-added services such as liquidity mining for users to experience a one-stop decentralized financial project.
```

What is the primary source of funding for this project?
```
Our own capital.
```

What other projects/ecosystem stakeholders is this project associated with?
```
Polkadot, Ethereum, etc.. We have made corresponding investments in several tracks.
```

## Use-case details

Describe the data being stored onto Filecoin
```
As an investment institution, we regularly conduct research on different projects, and produce research reports in the form of text, video and graphics. We want to store our reports data on the Filecoin network. 
```

Where was the data in this dataset sourced from?
```
It comes from our research on different projects.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
https://mp.weixin.qq.com/s/12ycpypQXpC_aImM4Tqt5Q、
https://mp.weixin.qq.com/s/TQZb3eIQM2Z89fQCWH7IGw
https://mp.weixin.qq.com/s/BHm-eNfNYK30YHT10W-hJg
https://www.youtube.com/watch?v=OfZE72co_m8
https://www.youtube.com/watch?v=5q0QMHr5fkw
https://www.youtube.com/watch?v=95eXF8kLE2M&t=19s
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
We confirm that is public dataset that can be retrieved by anyone on the network. 
```

What is the expected retrieval frequency for this data?
```
There is no fixed retrieval frequency of data retrieval.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
It would be better if it could be stored permanently.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
Asia, USA
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
About dozens
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
200TiB weekly
```

How will you be distributing deals across storage providers?
```
We prefer online transfer, but if there are any special circumstances, we also accept offline transfer. 
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Yes.
```

**Comment by galen-mcandrew** [comment]
## Multisig Notary requested
#### Total DataCap requested 
> 5PiB
#### Expected weekly DataCap usage rate 
> 100TiB

**Comment by dannyob** [comment]
Okay, this looks good enough to me for an initial provision. Please let us know how you found your storage providers and what data you're intending to store using your first allotment of datacap.

**Comment by dannyob** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecmjg4duvqbz3arr3har63x4tcyh6it3sk56xhi2xjnp4yc2cvdyg 
#### Address 
> f1dfkdmjhuvvol6okun57mix447wdpolwcd323ktq
#### Datacap Allocated
> 50TiB
#### Signer Address
> f1k6wwevxvp466ybil7y2scqlhtnrz5atjkkyvm4a
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecmjg4duvqbz3arr3har63x4tcyh6it3sk56xhi2xjnp4yc2cvdyg

**Comment by MegTei** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecvdsoftjmmewbpg2xgusg2lxit6hk4dnvchjzvb7r4wuvlst3luw 
#### Address 
> f1dfkdmjhuvvol6okun57mix447wdpolwcd323ktq
#### Datacap Allocated
> 50TiB
#### Signer Address
> f1ystxl2ootvpirpa7ebgwl7vlhwkbx2r4zjxwe5i
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecvdsoftjmmewbpg2xgusg2lxit6hk4dnvchjzvb7r4wuvlst3luw

**Comment by mr-spaghetti-code** [comment]
Hi,

It looks like you received 50TiBs of DataCap to date but have spent less than 20% of it so far.

We would love to understand if there's anything holding you back. We are working hard to make the data onboarding process easier for clients like you and your feedback is very valuable. If you have a moment, please fill in this survey: https://forms.gle/s6AuTXZPZSMokscLA

If you have any feedback or would like to consult with an expert, please let me know. 

Thanks,

João Fiadeiro
Product Manager, Large Data Client Onboarding
Protocol Labs

**Comment by ClarkYin5566** [comment]
Hi @Bitrise2020 , I am also applying for datacap but still waiting for review. I would like to discuss with you how to proceed and how to promote the development of real data. If possible, please leave your email or contact me: mr.clark.yin@gmail.com

**Comment by rayshitou** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecaqroidbrpgoq7jjcv3r5zcfpijzzlaws364o3ffws255ulyjtme 
#### Address 
> f1dfkdmjhuvvol6okun57mix447wdpolwcd323ktq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f136bz32l2hlyad4npmfg5dngcdw3jk3tpcpoz2ja
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecaqroidbrpgoq7jjcv3r5zcfpijzzlaws364o3ffws255ulyjtme

**Comment by swatchliu** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedycylyxkhcizsqeupxvexd32k6xwdsokrtko7entbfkndke2kes6 
#### Address 
> f1dfkdmjhuvvol6okun57mix447wdpolwcd323ktq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1yh6q3nmsg7i2sys7f7dexcuajgoweudcqj2chfi
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedycylyxkhcizsqeupxvexd32k6xwdsokrtko7entbfkndke2kes6

**Comment by cryptowhizzard** [comment]
Hello @Bitrise2020 

It seems you spend your datacap to 4 providers where none of the providers allow public retrieval of the data. This is against the FIL+ / LDN rules. Therefore I (we) cannot grant you further datacap unfortunately.

Please confirm that you are aware that the data stored should be publicly retrievable and that you are aware that it is your responsibility to check this and make this happen.

Do you intend to make a recovery program to re-upload the data to providers with good reputation and do allow retrieval?
If so, can you provide us a list of SP's to check?

-----------------------------------------------------

root@proposals:/var/www/html# lotus net connect f01715155
f01715155 -> {12D3KooWFcLVPs2bzj8Zz8eSbGob3Wzp4ufqd8Cy9MbwjWfHDR8k: [/ip4/10.81.6.4/tcp/39465]}
ERROR: failed to parse multiaddr "f01715155": must begin with /

root@proposals:/var/www/html# lotus net connect f01740934
f01740934 -> {12D3KooWG27MHa8xSeHg1vFM9d1hsDHrQdGRhWG7aQ2id1mhF5fM: []}
ERROR: failed to parse multiaddr "f01740934": must begin with /

root@proposals:/var/www/html# lotus net connect f01834253
f01834253 -> {12D3KooWJ1XnB9USErotNgHrP7eZv9C3RYpAAJio69MhYYA2aCsX: []}
ERROR: failed to parse multiaddr "f01834253": must begin with /

root@proposals:/var/www/html# lotus net connect f01834291
f01834291 -> {12D3KooWDoPHuPhcEh6cbXAzvowRdqkrza2xpAXXgJKy12hKsfVr: []}
ERROR: failed to parse multiaddr "f01834291": must begin with /

![Schermafbeelding 2022-06-06 om 15 23 22](https://user-images.githubusercontent.com/36404435/172169180-12572ed3-c4c2-4d7b-95ea-c78086a21c5c.png)

**Comment by Bitrise2020** [comment]
Hello, @cryptowhizzard 
We are very clear about the rules.

Allow me to explain the reasons for your concern:
Most data of Bitrise are come from Singapore and NA. Based on the situation for our entity, we found SPs from Singapore and America. HOWEVER, there are not many SPs disclosed in these two regions, and there are very few organizations that we can communicate with.
Therefore, in the early sealing, we selected several SPs that were relatively safe from our investigation. No matter in location or reputation. 
We hope the data can be allocated on trustable SPs. But we do not think that the SPs we used are unqualified SPs. We found that those SPs which can't be publicly retrieved just for security reasons. f01740934, f01715155 can be public retrieval now.
During this period, we added more constraints on the SP selection conditions. For example, f01850141 f01863339 are the new SPs we are ready to cooperate with. There are still some SPs communicating with us, if you need, I can list them after I communicate with each other.

We certainly hope to make a recovery program. In addition, we want to take this opportunity to cooperate with more SPs.

**Comment by StFileCloud** [comment]
> @Filefly-ph Thank god! Really pleased to see that someone can see us! I'm also wondering why no one notaries could reply to us and curious about the processing times of notaries within the community since we waited two months for this issue. Are you a notary? If so, can you communicate with us using the official GitHub account? We appreciate your contribution!

Yes, I'm a notary. As I mentioned before, some of your SPs went wrong months ago and I found there are still 2 SPs are not accessible therefore I asked for rootCID to check whether all data are retrievable and SPs are dialable. Once it's confirmed, we will restore the process of multisig. 
![image](https://user-images.githubusercontent.com/106466939/180449049-5aa7c79e-84cc-4fba-8bf7-e95622d5a469.png)

**Comment by Filefly-ph** [comment]
> @Filefly-ph Thank god! Really pleased to see that someone can see us! I'm also wondering why no one notaries could reply to us and curious about the processing times of notaries within the community since we waited two months for this issue. Are you a notary? If so, can you communicate with us using the official GitHub account? We appreciate your contribution!

Yes I'm a notary. As I've mentioned before, some of your SPs were not accessible months ago and I found there are still 2 SPs are not accessible therefore I asked for rootCID to check whether all data are retrievable. Once it's confirmed, we will restore the mulitsig process
![image](https://user-images.githubusercontent.com/106759901/180450377-8c7ef7ea-6c59-4efc-a41c-1f908b6c6ac5.png)

**Comment by Bitrise2020** [comment]
Hello, @Filefly-ph 
The CID is preparing by the tech team. Please be patient for a while. 
Is your account related to the notary @FireflyHZ? Can you provide proof of identity?

**Comment by fireflyHZ** [comment]
> Hello, @Filefly-ph The CID is preparing by the tech team. Please be patient for a while. Is your account related to the notary @fireflyHZ? Can you provide proof of identity?

em...em.. I don't know @Filefly-ph .
